volttron/platform/web/csr_endpoints.py

```diff
@@ -51,13 +63,11 @@ def _csr_request_new(self, env, data):
                      headers={'Content-type': 'application/json'})
 
         self._certs.save_pending_csr_request(env.get('REMOTE_ADDR'), identity, csr)
-        csr_file = self._certs.csr_pending_file(identity)
-        if csr:
-            with open(csr_file, "wb") as fw:
-                fw.write(csr)
 
-        auto_accept = True
+        auto_accept = False
         if auto_accept:
+            if self._certs.cert_exists(identity, True):
+                cert = self._certs.cert(identity, True)
```

Comment:
Line 76 any way create a new signed cert from csr so why load this cert?

Reply:
Currently no test cases...I will add to a later pull request.  If you want to test this you will need to vms that can connect to each other.  I have to make documentation for this as well today.  Again another pull request to come.